packaging/ubuntu/debian/control

```diff
@@ -20,6 +21,7 @@ Build-Depends: asciidoc,
                libusb-1.0-0-dev,
                libbluetooth-dev,
                libmtp-dev,
+               libraw-dev,
```

Comment:
I think we've doubled up here. 😆

Reply:
I didn't notice.
And you are adding the runtime dependency I notice, which again, I believe, isn't needed.
Obviously we should drop one of them (although the tools are smart enough not to care)